package/kernel/linux/modules/usb.mk

```diff
@@ -1231,6 +1231,22 @@ endef
 $(eval $(call KernelPackage,usb-net-mcs7830))
 
 
+define KernelPackage/usb-net-smsc75xx
+  TITLE:=SMSC LAN75XX based USB 2.0 Gigabit ethernet devices
+  DEPENDS:=+LINUX_5_10:kmod-libphy +!LINUX_5_4:kmod-libphy
```

Comment:
This `DEPENDS` line looks wrong to me.
I think it should look like this:
```
 DEPENDS:=+!LINUX_5_4:kmod-libphy
```

Reply:
Indeed, this was a misunderstanding of my earlier comment.

Reply:
fixed